mandatory/4-tax.js

```diff
@@ -17,7 +18,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  return `£${calculateSalesTax(price).toFixed(2)}`;
```

Comment:
Interpolation is generally a very cool technique to reduce the amount of code you write, but consider edge cases like this one - you have an unusual combination of `£$` that looks strange for the reader. For this specific example, I'd consider a variable to reduce "visual clutter" here and simplify the reader's job.